microsoft-edge/web-platform/user-agent-guidance.md

```diff
@@ -158,19 +156,19 @@ Wherever possible, Microsoft recommends minimizing use of Microsoft Edge browser
 
 For legacy reference, the following format was used for User-Agent string.
 
-On Windows, the `User-Agent` HTTP request header uses the following format.
+On Windows, the `User-Agent` HTTP request header uses the following format:
 
 ```https
 User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/90.0.4430.85 Safari/537.36 Edg/90.0.818.46
 ```
 
-On Android, the `User-Agent` HTTP request header uses the following format.
+On Android, the `User-Agent` HTTP request header uses the following format:
 
 ```https
-User-Agent: Mozilla/5.0 (Linux; Android 6.0; Nexus 5 Build/MRA58N) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/90.0.4430.85 Mobile Safari/537.36 Edg/90.0.818.46
+User-Agent: Mozilla/5.0 (Linux; Android 6.0; Nexus 5 Build/MRA58N) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/90.0.4430.85 Mobile Safari/537.36 EdgA/90.0.818.46

```

Comment:
This change is the motivation for this PR.

Reply:
Do we need to explicitly provide a full Android UA string here, or is the desktop sufficient given we have the table with potential Edge tokens below? If we wanna keep this, then LGTM.